packages/swingset-liveslots/test/virtual-objects/test-cross-facet.js

```diff
@@ -33,10 +33,12 @@ test('forbid cross-facet prototype attack', t => {
   thing2.mutable.set(2);
 
   t.throws(() => attack1(thing1.mutable, thing2.immutable), {
-    message: /^illegal cross-facet access/,
+    message:
+      '"In \\"set\\" method of (thing mutable)" may only be applied to a valid instance: "[Alleged: thing immutable]"',
   });
   t.throws(() => attack2(thing1.mutable, thing2.immutable), {
-    message: /^illegal cross-facet access/,
+    message:
+      '"In \\"set\\" method of (thing mutable)" may only be applied to a valid instance: "[Alleged: thing immutable]"',
```

Comment:
This test-case change demonstrates what should be the only observable difference of changing how contextProviders report a miss: from throwing to returning `undefined`. The `undefined` is then turned into a thrown error at a higher level which is able to generate a better diagnostic.

Reply:
:100: